### ISSUE TITLE
Will NSCA-ng support icinga2 after the 'External Command Pipe' is removed?

### ISSUE THREAD

**Issue by isaaclw** [weiss/nsca-ng#7]
On the road map, it looks like icinga2 is planning to deprecate the [External Command Pipe](https://icinga.com/docs/icinga-2/latest/doc/14-features/#external-command-pipe).

If this were to change, would nsca-ng continue to support icinga2, via API or some other method?

Note: https://github.com/Icinga/icinga2/issues/6307

Thanks for your work on this.

**Comment by weiss** [comment]
Yes, that would make sense.  I might add support at some point in the future, but can't make promises, sorry.

I'll keep this issue open.
